src/main/cpp/util/file_windows.cc

```diff
@@ -259,7 +259,15 @@ bool ReadFile(const string& filename, string* content, int max_size) {
     content->clear();
     return true;
   }
-  return ReadFile(Path(filename), content, max_size);
+
+  // In order for try-imports to be ignored gracefully we need to check for an error with the path and return false
+  // rather than die.
+  std::string errorText;
+  Path path = Path(filename, &errorText);
+  if (!errorText.empty()) {
+      return false;
```

Comment:
Maybe we can print the error as a warning here?
See https://cs.opensource.google/bazel/bazel/+/master:src/main/cpp/util/logging.h;l=101;bpv=1;bpt=1

Reply:
Added a warning